components/AddForm.js

```diff
@@ -0,0 +1,70 @@
+import CheckIcon from '../public/assets/check.svg';
+import styled from 'styled-components';
+
+export default function AddForm({ onAddItem, setAddActive }) {
```

Comment:
See my remark in the Footer component, you could avoid passing the state here, by just maintaining the state here. 
Another option is to maintain in the index.js, then you would still need to pass the state as you are doing here :), i would prefer the first option though :)